### PR TITLE
Prepare ext:* commands for v11 with breaking changes

### PR DESCRIPTION
**Pull request by elvisun**
### Description

Prepare ext:* commands for v11 with breaking changes

**When reviewing please turn on the "hide whitespaces" setting so the diff is clear** (https://screenshot.googleplex.com/BqzNRwoa3WGKtwv)

DO NOT SUBMIT until v11 is ready

### CODE REVIEW

**Comment by joehan** [comment]
I believe we are using the 'next' branch to stage changes for v11. Want to change this to point at that branch?

File: src/commands/ext-configure.ts [hunk 1]
```diff
@@ -47,156 +43,72 @@ export default new Command("ext:configure <extensionInstanceId>")
...
-      if (isLocalSource) {
-        const source = await createSourceFromLocation(needProjectId({ projectId }), refOrPath);
-        spec = source.spec;
-      } else {
-        const extensionVersion = await extensionsApi.getExtensionVersion(refOrPath);
-        spec = extensionVersion.spec;
-      }
+    if (options.nonInteractive) {
+      throw new FirebaseError(
+        `Command not supported in non-interactive mode, edit ./extensions/${instanceId}.env directly instead`
```
Comment (joehan) [inline src/commands/ext-configure.ts:48]: Hmmm, if a user is hitting this error now, they are probably trying to figure out how to do noninteractive deploys. Consider including a manifest docs link to help them figure out the new way to do this?

**Comment by elvisun** [comment]
> I believe we are using the 'next' branch to stage changes for v11. Want to change this to point at that branch?

done

File: src/extensions/manifest.ts [hunk 2]
```diff
@@ -264,33 +264,17 @@ function readParamsFile(projectDir: string, fileName: string): Record<string, st
...
- * Show preview warning about --local flag needing deploy to take effect in firebase project.
- */
-export function showPreviewWarning() {
-  utils.logLabeledWarning(
-    logPrefix,
-    `See these changes in your Firebase Emulator by running "firebase emulators:start". ` +
-      `Run ${clc.bold(
-        "firebase deploy (--only extensions)"
-      )} to deploy the changes to your Firebase project. `
+      )} will deploy the changes to your Firebase project. `
```
Comment (joehan) [inline src/extensions/manifest.ts:278]: Add manifest docs link here? Something like "See (link) for more details.

**Review by joehan — APPROVED** [review]
Beautiful to see so much code cleaned up 💯

**Pull request merged**